From 5090a2475a456228e09519661d542be7dccb03d6 Mon Sep 17 00:00:00 2001
From: Sean Paul <seanpaul@chromium.org>
Date: Tue, 30 Jan 2018 09:47:01 -0500
Subject: [PATCH 1017/2345] UPSTREAM: drm/i915: Downgrade hdcp logs from INFO
 to DEBUG_KMS

HDCP was a bit too chatty to get along with the rest of the i915 driver.

Suggested-by: Daniel Vetter <daniel.vetter@ffwll.ch>
Reviewed-by: Daniel Vetter <daniel.vetter@ffwll.ch>
Signed-off-by: Sean Paul <seanpaul@chromium.org>
Link: https://patchwork.freedesktop.org/patch/msgid/20180130144707.171657-1-seanpaul@chromium.org
(cherry picked from commit b34e650224576502ffd5d71d03b6e28a5d0b6844)

Change-Id: Id890841608e3bde1637eec15c6109624e3128049
---
 drivers/gpu/drm/i915/intel_hdcp.c | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_hdcp.c b/drivers/gpu/drm/i915/intel_hdcp.c
index 9fc9da0..b97184e 100644
--- a/drivers/gpu/drm/i915/intel_hdcp.c
+++ b/drivers/gpu/drm/i915/intel_hdcp.c
@@ -379,7 +379,8 @@ int intel_hdcp_auth_downstream(struct intel_digital_port *intel_dig_port,
 		return -ENXIO;
 	}
 
-	DRM_INFO("HDCP is enabled (%d downstream devices)\n", num_downstream);
+	DRM_DEBUG_KMS("HDCP is enabled (%d downstream devices)\n",
+		      num_downstream);
 	return 0;
 }
 
@@ -500,7 +501,7 @@ static int intel_hdcp_auth(struct intel_digital_port *intel_dig_port,
 	if (repeater_present)
 		return intel_hdcp_auth_downstream(intel_dig_port, shim);
 
-	DRM_INFO("HDCP is enabled (no repeater present)\n");
+	DRM_DEBUG_KMS("HDCP is enabled (no repeater present)\n");
 	return 0;
 }
 
@@ -532,7 +533,7 @@ static int _intel_hdcp_disable(struct intel_connector *connector)
 		return ret;
 	}
 
-	DRM_INFO("HDCP is disabled\n");
+	DRM_DEBUG_KMS("HDCP is disabled\n");
 	return 0;
 }
 
@@ -735,7 +736,7 @@ int intel_hdcp_check_link(struct intel_connector *connector)
 		goto out;
 	}
 
-	DRM_INFO("HDCP link failed, retrying authentication\n");
+	DRM_DEBUG_KMS("HDCP link failed, retrying authentication\n");
 
 	ret = _intel_hdcp_disable(connector);
 	if (ret) {
-- 
2.7.4

